include/sys/vdev_disk.h

```diff
@@ -42,13 +42,5 @@
 
 #ifdef _KERNEL
 #include <sys/vdev.h>
-
-typedef struct vdev_disk {
-	ddi_devid_t		vd_devid;
-	char			*vd_minor;
-	struct block_device	*vd_bdev;
```

Comment:
I'm a little surprised this ever worked on FreeBSD 🤔

Reply:
I don't think FreeBSD even includes the header which is why it was OK.